packages/backend/src/dbt/dbtChildProcess.ts

```diff
@@ -75,8 +90,16 @@ export class DbtChildProcess {
                 dbtArgs.push('--target', this.target);
             }
 
+            if (this.profileName) {
+                dbtArgs.push('--profile', this.profileName);
+            }
+
             this.dbtChildProcess = execa('dbt', dbtArgs, {
                 stdio: ['pipe', 'pipe', process.stderr],
+                env: {
+                    ...process.env,
```

Comment:
why do we need to expose our envs ? sounds dangerous

Reply:
Good point.

The default behaviour is to pass process.env to the subprocess (i.e. this was already happening). I thought maybe there was something going on with $PATH. But I'll try without! I agree it would be better